packages/plugin-ext/src/main/browser/documents-main.ts

```diff
@@ -54,7 +54,8 @@ export class DocumentsMainImpl implements DocumentsMain {
             this.proxy.$acceptModelSaved(m.textEditorModel.uri);
         }));
         this.toDispose.push(modelService.onModelWillSave(onWillSaveModelEvent => {
-            onWillSaveModelEvent.waitUntil(new Promise<monaco.editor.IIdentifiedSingleEditOperation[]>(async resolve => {
+            onWillSaveModelEvent.waitUntil(new Promise<monaco.editor.IIdentifiedSingleEditOperation[]>(async (resolve, reject) => {
+                setTimeout(() => reject(new Error('Aborted onWillSaveTextDocument-event after 1750ms')), 1750);
```

Comment:
Hello, please extract 1750 to variable so in error message is synched with value of timeout ?

Also does-it make sense to have that timeout overridable ?